MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include CONTRIBUTING.md
+include LICENSE
+include README.md
+
+recursive-include tests *
+recursive-exclude * __pycache__
+recursive-exclude * *.py[co]
+
+recursive-include docs *.rst conf.py Makefile make.bat *.jpg *.png *.gif
```

Comment:
Some of these seem content-wise different from each other, maybe it's best to have those on separate lines?
```suggestion
recursive-include docs *.rst
recursive-include conf.py Makefile make.bat
recursive-include *.jpg *.png *.gif
```

Reply:
This Manifest.in was on the original cookiecutter-pypackage from the original maintainers. Don't want to mess with it too much but will tinker when I can.